python/common/org/python/types/CaseFolding.txt

```diff
@@ -0,0 +1,104 @@
+00DF; F; 0073 0073; # LATIN SMALL LETTER SHARP S
```

Comment:
What is the source of this data? It strikes me as unusual that this (a) is all the entries required in a case folding table, and (b) that Java doesn't have such a table internally.